crates/bevy_ui/src/render/mod.rs

```diff
@@ -487,51 +353,14 @@ pub fn extract_uinode_borders(
 
         // Skip invisible borders
         if !view_visibility.get()
-            || style.display == Display::None
             || maybe_border_color.is_some_and(|border_color| border_color.0.is_fully_transparent())
```

Comment:
Looks unrelated

Reply:
Removing this check means we don't have to query for `Style`, With  `Display::None` the thickness is zero so it still gets skipped.